webapp/src/utils/fileUtils.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Triggers the download of a file with the given data and name.
+ *
+ * @param fileData - The data of the file to be downloaded.
+ * @param fileName - The name of the file to be downloaded.
+ */
+export function downloadFile(fileData: BlobPart, fileName: string) {
```

Comment:
rename `processDownload` or something else and move it to the utils file of `DownloadButton`